vtex-apps/store-components
# fix: update sku selector on plp

#### What problem is this solving?



Fix to the KI 341322 ( https://vtexhelp.zendesk.com/agent/tickets/341322 ) update SKU selector in plp after change SKU in quickview

#### How to test it?

you could test the fix in this workspace 

https://vtexsku--partnerspruebascol.myvtex.com/apparel/sony?initialMap=c&initialQuery=apparel&map=category-1,brand

1. Go to any store that has a quick view and sku-selector.
2. Pick an SKU and go to the quick view mode
3. Change the SKU inside the quick view modal
4. Exit quick view mode and you will notice that the sku didn't change, it just changed inside the quick view context

[Workspace](https://vtexsku--partnerspruebascol.myvtex.com/apparel/sony?initialMap=c&initialQuery=apparel&map=category-1,brand)

#### Screenshots or example usage:

https://www.loom.com/share/f74bcda6646c4d89a660b4f2a2f0a6f5?sid=b4134cb1-3c2b-440b-b69e-e58c6cd6704a

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExY2Vocjc2N3hrNmR0NW0yNW44OWlqZWxqZXQyMjN1c3BpMWExd3p4aiZlcD12MV9naWZzX3RyZW5kaW5nJmN0PWc/Dg4TxjYikCpiGd7tYs/giphy.gif)
